semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtilsTest.java

```diff
@@ -294,6 +299,27 @@ public void testGetHttpScheme() {
 
     event = createMockEventWithAttribute(OTelHttpSemanticConventions.HTTP_SCHEME.getValue(), "");
     assertTrue(HttpSemanticConventionUtils.getHttpScheme(event).isEmpty());
+
+    event = createMockEventWithAttribute(HTTP_REQUEST_X_FORWARDED_PROTO, "http");
+    assertEquals(Optional.of("http"), HttpSemanticConventionUtils.getHttpScheme(event));
+
+    event =
+        createMockEventWithAttribute(HTTP_REQUEST_FORWARDED, "by=random;proto=https;for=modnar");
```

Comment:
Add one test w/o having `proto` field. eg. `by=random;for=modnar` and with spaces `by=random; proto=https; for=modnar`

Reply:
added